website/docs/commands/export.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Export ⚡️
+sidebar_position: 27
+---
+
+The `export` sub-command outputs information about the current project to a format used by external build tools such as SBT or Mill.
+Additionally the command supports the JSON format for custom analysis of projects.
```

Comment:
```suggestion
The `export` sub-command converts a Scala CLI project into an SBT or Mill project configuration. When your simple project starts to grow and you need support for modules, it would be useful to use a build tool such as SBT or Mill.
Additionally the `export` sub-command supports the JSON format for custom analysis of projects.
```